scripts/utils/events.js

```diff
@@ -56,11 +56,19 @@ const projectsEvents = () => {
   projectForm.addEventListener("submit", projectsFormSubmit);
 };
 
-
-
-
-const overviewEvents = () => {
-  return;
+const overviewEvents = (e) => {
+  e.preventDefault()
+  if(e.target.id === 'pinRepoForm'){
```

Comment:
If the pin form is submitted, it pins the repo in the array. Because the card and the form depend on that data, I'm rerendering both so that they can be updated.